third_party/move/move-compiler-v2/tests/reference-safety/multiple_use_invalid.exp

```diff
@@ -28,3 +28,13 @@ error: same mutable reference in local `r1` is also used in other local `r2` in
    │                  ------ previous mutable local borrow
 20 │         s(r1, r2)
    │         ^^^^^^^^^ requirement enforced here
+
```

Comment:
V1 does not generate this error

Reply:
Is this already a v1 test or can we add it to the v1 test tree?

Reply:
I did not find the test file with this name in V1

Reply:
Yes, this test now fails after the modification of the freeze semantics in v2. I think its a rare case so we are ok with that (urgent issue would be it would succeed but fail in the bytecode verifier). See also below about the weirdness of freeze semantics.